<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedQuizWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageQuizMenu">
       <widget class="QPushButton" name="pushButtonLoad">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>450</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>450</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonStudyPage">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>450</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Study</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCreatePage">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>450</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageQuizCreate">
       <widget class="QLineEdit" name="lineEditQuestion">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>40</y>
          <width>611</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnswer5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>360</y>
          <width>611</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnswer1">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>120</y>
          <width>611</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnswer5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>360</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Answer</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnswer1">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Answer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnswer2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>180</y>
          <width>611</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnswer4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>300</y>
          <width>611</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnswer4">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>300</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Answer</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelQuestion">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Question</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnswer3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>240</y>
          <width>611</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditAnswer6">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>420</y>
          <width>611</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnswer6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>420</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Answer</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnswer3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>240</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Answer</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAnswer2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert Answer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCreateQuestion">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>500</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonOverwriteQuestion">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>500</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Overwrite</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonDeleteQuestion">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>500</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPreviousQuestion">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>500</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonNextQuestion">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>500</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonReturn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxCorrect">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>460</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="labelCorrectAnswer">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>460</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Correct Answer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageQuizStudy">
       <widget class="QPushButton" name="pushButtonReturn_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelDisplayQuestion">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Question:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayQuestion">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>69</y>
          <width>681</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAnswer1">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>160</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAnswer2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>210</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAnswer4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>310</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAnswer3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>260</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAnswer6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>410</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAnswer5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>360</y>
          <width>90</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPreviousQuestion_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>490</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonNextQuestion_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>490</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonSubmitQuiz">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>490</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayAnswer1">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayAnswer2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>200</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayAnswer4">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>300</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayAnswer3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>250</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayAnswer6">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>400</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textDisplayAnswer5">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>350</y>
          <width>561</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageQuizResults">
       <widget class="QPushButton" name="pushButtonReturn_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelResult">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textResultNum">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>104</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textResultPer">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>104</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelPercentSign">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>170</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelFractionSign">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>130</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="textResultDen">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>120</y>
          <width>104</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
